packages/components/organisms/f-header/src/components/CountrySelector.vue

```diff
@@ -0,0 +1,288 @@
+<template>
+    <li
+        :class="['c-nav-list-item has-sublist', {
+            'open': countrySelectorIsOpen
+        }]"
+        data-test-id="countrySelector-toggle"
+        v-on="isBelowMid ? null : { mouseover: openCountrySelector, mouseleave: closeCountrySelector }"
+        @keyup.esc="closeCountrySelector">
+        <f-button
+            button-type="icon"
+            class="c-nav-list-text c-country-selector-list-item"
+            :aria-expanded="!isBelowMid && countrySelectorIsOpen ? 'true' : 'false'"
+            :aria-haspopup="isBelowMid ? false : true"
+            :aria-label="changeCountryText"
+            @click.native="onCountrySelectorToggle"
+            v-on="isBelowMid ? { blur: closeNav, focus: openNav } : null">
+            <span class="c-countrySelector-currentFlag-wrapper">
+                <flag-icon
+                    :country-code="currentCountryKey"
+                    class="c-countrySelector-flag c-countrySelector-flag--current" />
+            </span>
+            <span class='c-countrySelector-title'>
+                {{ selectYourCountryText }}
+            </span>
+        </f-button>
+
+        <div :class="['c-nav-panel', { 'is-visible': countrySelectorIsOpen }]">
+            <div class="c-nav-popoverList c-nav-popoverList--countrySelector">
```

Comment:
Out of interest – does this reuse the styling and functionality of the other popover list? If so, it may be worth either:

1. Extracting out the component if it's similar enough, or
2. extracting out the styles into `common.scss` so that we aren't duplicating styles between here and the other popoverList styling

Reply:
@ashleynolan it reuses some styles. I can make a separate ticket for this if you don't mind because we have a tight timeline for this to go live as part of "new old homepage". All works as expected so it shouldn't block us from releasing it.